client/packages/common/src/ui/forms/JsonForms/components/Text.tsx

```diff
@@ -51,7 +54,8 @@ const UIComponent = (props: ControlProps) => {
         },
         disabled: !props.enabled,
         error,
-        helperText: errors,
+        helperText:
+          errors && examples ? `Use e.g.: "${examples.join('", "')}"` : errors,
```

Comment:
I used the short Use e.g. because there where some weird spacing when the error became to long. But this should really be solved somewhere else. Will change and make it translatable, thanks!